tests/steps/build_step/sbt/test_sbt.py

```diff
@@ -0,0 +1,14 @@
+from src.mpyl.steps.build.sbt import BuildSbt
+from src.mpyl.steps.models import Input
+from tests.test_resources import test_data
+from tests.test_resources.test_data import get_project
+
+
+class TestBuildSbt:
```

Comment:
In the future we could convert these tests to use the [unittest](`unittest`) library, like what DA is using in their scripting repo, if we feel like we need more facilities for testing

Reply:
Oh, I actually moved away from the standard  python `unittest` library :-) https://github.com/Vandebron/mpyl/pull/12

Reply:
Ohh I should've checked.. Then disregard my comment!